source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -0,0 +1,409 @@
+.. _node-fundamentals-bulk:
+
+===============
+Bulk Operations
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to
+perform **bulk operations**.
+
+Bulk operations perform multiple write operations against one or more
+collections within a database. In MongoDB, you refer to a collection by combining
+the database name and the collection name in the format ``<database>.<collection>``. 
+Since you perform bulk operations on a ``MongoClient`` instance, you can perform
+bulk operations against any database and collection in the cluster accessed by 
+your client.
+
+Bulk operations help reduce the number of calls to the server. Instead of sending
+a request for each operation, you can perform multiple operations within one action.
+
+This guide includes the following sections:
+
+- :ref:`node-bulk-sample-data` presents the sample data that is used by the
+  bulk operation examples.
+- :ref:`node-bulk-operation-types` describes how to use different bulk operation 
+  types to perform bulk insert, replace, update, and delete operations.
+- :ref:`node-bulk-return-type` describes the return object that results from your
+  bulk write operations.
+- :ref:`node-bulk-handle-exceptions` describes the exceptions that occur if 
+  any of the operations in a bulk write operation fail.
+- :ref:`node-bulk-addtl-info` provides links to resources and API documentation for
+  types and methods mentioned in this guide.
+
+.. important::
+
+   To perform bulk write operations, ensure that your application meets the
+   following requirements:
+
+   - You are connected to MongoDB Server version 3.2 or later.
+   - You are using {+driver-short+} version 3.6 or later.
+
+.. _node-bulk-sample-data:
+
+Sample data
+-----------
+
+The examples in this guide use the following sample documents, which are stored
+in the ``movies`` collection in the ``sample_mflix`` database:
+
+.. literalinclude:: /code-snippets/crud/bulk.js
+   :language: javascript
+   :dedent:
+   :start-after: begin-sample-data
+   :end-before: end-sample-data 
+
+.. _node-bulk-operation-types:
```

Comment:
Either in this section or in each of the operation type examples, we should show the assignment of the `movies` variable in the example code to clarify to users that `movies` is a `Collection` object.